graylog2-web-interface/src/views/components/searchbar/saved-search/SavedSearchList.jsx

```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { browserHistory } from 'react-router';
+// $FlowFixMe should be there
```

Comment:
Looks like this comment is not required

Reply:
Well my flow does say something else :shrug: